### PR TITLE
updated national help line number

### PR DESCRIPTION
**Pull request by GalacticMaster**
I am using your data in one of our bot. We are a group of Mumbai based techi, govt officials and social workers. 

I am updating contacts API for national help line number.

### CODE REVIEW

**Comment by GalacticMaster** [comment]
updated national help line numbers

**Comment by amodm** [comment]
Source? Helpline as mentioned [here](https://www.mohfw.gov.in) is the one that's currently mentioned

**Comment by GalacticMaster** [comment]
http://newsonair.nic.in/News?title=Toll-free-24x7-national-helpline-number-1075-to-address-queries-related-to-COVID-19%3B-Total-confirmed-cases-rise-to-114-in-India&id=383104

**Comment by amodm** [comment]
In order to maintain consistency of schema, can you modify this to:
```
"number": "+91-11-23978046",
"email": "ncov2019@gmail.com",
"twitter": "https://twitter.com/MoHFW_INDIA",
"others": [
  {"helpline": "1075", "source": "https://..."},
  {"helpline": "1800-112-545", "source": "https://..."}
]
```

**Comment by amodm** [comment]
Idea behind capturing `source` is to ensure that MOHFW's published data is tracked separately from any other sources.

File: data/contacts-in.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
     "primary": {
+        "national help line": "1075",
+        "national help line": "1800-112-545",
```
Comment (amodm) [inline data/contacts-in.json]: Remove this as not showing up in official website

File: data/contacts-in.json [hunk 2]
```diff
@@ -1,5 +1,7 @@
 {
     "primary": {
+        "national help line": "1075",
```
Comment (amodm) [inline data/contacts-in.json]: Can you rename this to "number-tollfree"? This is now showing up in official website

**Comment by amodm** [comment]
In the interest of time, I've made the changes myself, and noted your contributions in README

**Pull request closed without merging**